docker/candlepin-base-cs8/setup-devel-env.sh

```diff
@@ -61,23 +61,6 @@ BASHRC
 git clone https://github.com/candlepin/candlepin.git /candlepin
 cd /candlepin
 
-# Setup and install rvm, ruby and pals
```

Comment:
We can't remove ruby from our c7 and cs8 containers. We need it to run rspec in older branches.